app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -6,10 +6,16 @@ class ApplicationFormController < CandidateInterfaceController
     def show
       @application_form_presenter = CandidateInterface::ApplicationFormPresenter.new(current_application)
       @application_form = current_application
+      @apply_1_application_form = current_candidate.application_forms.apply_1.last
```

Comment:
To avoid coupling applying again and apply 1/2, we can make this the previous application: 

```suggestion
      @previous_application_form = current_candidate.application_forms.where.not(id: current_application.id).order(id: :desc).last
```

https://ukgovernmentdfe.slack.com/archives/CP18YJXPY/p1588250617304800

Reply:
have addressed in the other PR